config/locales/api/hi_IN.yml

```diff
@@ -1,44 +1,44 @@
+      invalid_password: "आपका पासवर्ड मेल नहीं खा रहा। फिर प्रयास करना चाहेंगे?"
       account_locked: "आपका खाता अगले %{minutes} मिनटों के लिए लॉक कर दिया गया है। कृपया प्रतीक्षा करें और फिर से प्रयास करें।"
   sms:
-    request_otp: '<#> %{otp} आपका सरल सत्यापन कोड है\n%{app_signature}'
+    request_otp: "<#> %{otp} आपका सरल सत्यापन कोड है\n%{app_signature}"
     appointment_reminders:
-      missed_visit_sms_reminder: '%{facility_name} के हमारे कर्मचारी आपके और आपके दिल के स्वास्थ्य का ख्याल रखते हैं। कृपया अपने रक्तचाप की दवाइयाँ जारी रखें। नज़दीकी उप केंद्र से अपनी दवाई ले लें। अपने ANM या ASHA से संपर्क करें।'
-      missed_visit_whatsapp_reminder: '%{facility_name} के हमारे कर्मचारी आपके और आपके दिल के स्वास्थ्य का ख्याल रखते हैं। कृपया अपने रक्तचाप की दवाइयाँ जारी रखें। नज़दीकी उप केंद्र से अपनी दवाई ले लें। अपने ANM या ASHA से संपर्क करें।'
-    patient_request_otp: '<#> %{otp} आपका BP पासपोर्ट सत्यापन कोड है\n%{app_signature}'
+      missed_visit_sms_reminder: "%{facility_name} के हमारे कर्मचारी आपके और आपके दिल के स्वास्थ्य का ख्याल रखते हैं। कृपया अपने रक्तचाप की दवाइयाँ जारी रखें। नज़दीकी उप केंद्र से अपनी दवाई ले लें। अपने ANM या ASHA से संपर्क करें।"
+      missed_visit_whatsapp_reminder: "%{facility_name} के हमारे कर्मचारी आपके और आपके दिल के स्वास्थ्य का ख्याल रखते हैं। कृपया अपने रक्तचाप की दवाइयाँ जारी रखें। नज़दीकी उप केंद्र से अपनी दवाई ले लें। अपने ANM या ASHA से संपर्क करें।"
+    patient_request_otp: "<#> %{otp} आपका BP पासपोर्ट सत्यापन कोड है\n%{app_signature}"
   analytics:
     followup_patients: "कितने फॉलो-अप मरीज %{facility_name} गए?"
     patients_enrolled: "कितने मरीजों ने %{facility_name} पर नामांकन किया?"
     all_time_patients: "सभी समय: %{count} मरीज़"
     updated_at: "%{time} पर अद्यतित"
   user_analytics:
-    hypertension: 'हाइपरटेंशन'
-    diabetes: 'मधुमेह'
-    genders: ['सभी', पुरुष', 'महिला', 'ट्रांसजेंडर']
-    all_time: 'हमेशा'
-    follow_up_patients: 'हाइपरटेंशन के रोगियों को फ़ॉलो-अप करें'
-    registered_patients: 'रजिस्टर किए गए रोगी'
-    hypertension_control: 'हाइपरटेंशन का नियंत्रण'
-    hypertension_control_desc: 'माह की पिछली फ़ॉलो-अप विज़िट में हाइपरटेंशन के कितने रोगियों का बीपी 140/90 से कम था?'
-    notes: 'नोट'
-    footer_note_1: '%{facility_name} की सभी गतिविधि का डेटा है। अगर डेटा पुराना दिखाई दे रहा है, तो होम स्क्रीन पर "लंबित को सिंक्रनाइज़ करें" लिंक पर टैप करें।'
-    footer_note_2: 'फ़ॉलो अप रोगियों को हर माह केवल एक विज़िट के लिए गिना जाता है। अगर कोई रोगी, एक माह में एक से ज़्यादा बार फ़ॉलो अप करता है, तो उसे केवल एक बार ही गिना जाएगा।'
-    thank_you: 'आपका धन्यवाद'
-    thank_you_note: 'हाइपरटेंशन के रोगियों का उपचार करने में आपकी कड़ी मेहनत के लिए धन्यवाद। आपका काम, हृदयाघात और स्ट्रोक से लोगों का जीवन बचाता है।'
-    updated: 'अपडेट किया गया: %{time}'
-    ratio_string: '%{value_2} का %{value_1}'
+    hypertension: "हाइपरटेंशन"
+    diabetes: "मधुमेह"
+    genders: ["सभी", "पुरुष", "महिला", "ट्रांसजेंडर"]
```

Comment:
This missing quote is problematic in many locale files under `api` (same goes for double-quotes). For eg; [api/en.yml](https://github.com/simpledotorg/simple-server/blob/master/config/locales/api/en.yml#L29).

Reply:
Oh, my bad. I thought only this guy had it. I'll tack on some additional commits 👍